spec/controllers/api/faqs_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Api::FaqsController, type: :controller do
```

Comment:
Can I ask for some tests for creating (with default and custom env values) and updating FAQs?